public_html/wp-content/mu-plugins/camptix-tweaks/addons/class-payment-options.php

```diff
@@ -201,41 +224,37 @@ private function only_one_payment_method( $payment_methods ) {
 	 * Renders all other payment methods except stripe
 	 *
 	 * @param array  $payment_methods
-	 * @param string $selected_payment_method Pre selected payment method
+	 * @param string $selected_payment_method Pre selected payment method.
 	 */
-	function render_alternate_payment_options( $payment_methods, $selected_payment_method ) {
+	public function render_alternate_payment_options( $payment_methods, $selected_payment_method ) {
 		if ( $this->only_one_payment_method( $payment_methods ) ) {
 			// bail if only one payment option is available.
 			return;
 		}
 
-		// Pre-select first payment method if stripe is not available
+		// Pre-select first payment method if stripe is not available.
 		if ( ! $this->is_stripe_available( $payment_methods ) && ! isset( $selected_payment_method ) ) {
 			$selected_payment_method = array_keys( $payment_methods )[0];
 		}
 
-		foreach ( $payment_methods as $payment_method_key => $payment_method ) {
-			if ( 'stripe' === $payment_method_key ) {
-				continue;
-			}
```

Comment:
Why also include Stripe in the alternatives list when it already has its own button?

Reply:
Well, it's a list of all payment options, and it might not be immediately clear that you can click the stripe button again.

There's more potential for confusion for a screen reader user — reading through the list of "Payment Methods", it's more straightforward to select stripe from this list of options, vs knowing that they need to leave the list, and reverse-navigate back to the first toggle button.

Reply:
That makes sense. What if we changed the label to "All payment methods" or something? It just seems confusing to me to see "other" and then have "Credit Card" be listed in there again.

Reply:
"All payment methods" sounds good to me 👍